### ISSUE TITLE
It's hard to follow the issue title in planner on work item page 

### ISSUE THREAD

**Issue by hrishin** [openshiftio/openshift.io#3931]
As a user, if I navigate to item detail page it's hard to follow an item/issue title. Also, the box around title it's not comfortable to read.

e.g.
![planner_withbox](https://user-images.githubusercontent.com/28615674/42306796-012b77e4-804e-11e8-9cec-808485c77f5a.png)

Would it be better to change title format? 

for instance
![with_bold_lines](https://user-images.githubusercontent.com/28615674/42306925-6f74a586-804e-11e8-8214-3272b1fa3176.png)

**Comment by hrishin** [comment]
@jarifibrahim thanks!

**Comment by serenamarie125** [comment]
Based on a conversation held on 2019-Jan-04 including Dev, PM and UX, the UX team will not be providing support for planner items. Closing this issue since this was solely around gathering requirements.
